packages/commons-ui-core/src/ImageButton/ImageButton.js

```diff
@@ -38,10 +38,10 @@ const ImageRoot = styled("span", {
 }));
 
 const ImageButton = React.forwardRef(function ImageButton(props, ref) {
-  const { src } = props;
-  let image;
+  const { children, src, onClick } = props;
+  let image = children;
   if (src) {
-    image = <ImageRoot src={src} />;
+    image = <ImageRoot src={src} onClick={onClick} />;
```

Comment:
```suggestion
    image = <ImageRoot src={src} />;
```
What we want to be clickable is the button and not `ImageRoot`.